static/src/javascripts/bootstraps/enhanced/media/main.js

```diff
@@ -326,7 +326,9 @@ define([
                         resolve();
                     }
 
-                    if (abVideoAutoplay.variant === 'autoplay') {
+                    var contentType = config.page.contentType;
+                    var isArticleOrLiveBlog = contentType === 'Article' || contentType === 'LiveBlog';
+                    if (abVideoAutoplay.variant === 'autoplay' && isArticleOrLiveBlog) {
```

Comment:
if it's just for user testing/messing around then don't worry too much, however when you actually want to use the test data you have to be more careful as it's easy to much up the samples.  A good tip is to rename the test before you do the official testing so it doesn't pick up any data from people who were bucketed by old versions of the code.